.github/workflows/main.yml

```diff
@@ -27,82 +27,120 @@ jobs:
     steps:
     - name: Checkout Code
       uses: actions/checkout@v2
-
-    - name: Checkout Submodules
-      run: git submodule update --init --recursive
+      with:
+        submodules: 'recursive'
 
     - name: Install Python
       uses: actions/setup-python@v2
       with:
         python-version: '3.x'
 
-    - name: Install Build Tools (pip)
-      run: pip install meson ninja zstandard
-
     - name: Install Build Tools (apt)
-      run: sudo apt update && sudo apt install build-essential libsdl2-dev libogg-dev libopusfile-dev libpng-dev libzip-dev libx11-dev libwayland-dev python3-docutils libwebp-dev libfreetype6-dev libzstd-dev
-
-    - name: Configure - Step 1 (Meson)
-      run: meson setup --prefix=$(pwd)/build-test --wrap-mode=nofallback --force-fallback-for=cglm build/ -Db_lto=false -Dstrip=false -Ddeprecation_warnings=no-error -Dwerror=true -Dbasis_universal:werror=false -Db_pch=false
-
-    - name: Build - Step 2 (Ninja)
-      run: meson compile -C build/
-
-    - name: Install - Step 3 (Meson)
-      run: meson install -C build/
-
-    - name: Run - Step 4 (Taisei)
-      run: $(pwd)/build-test/bin/taisei -R $(pwd)/misc/tests/test-replay.tsr
+      run: |
+        sudo apt update
+        sudo apt install -y -qq build-essential libsdl2-dev libogg-dev libopusfile-dev libpng-dev libzip-dev libx11-dev libwayland-dev python3-docutils libwebp-dev libfreetype6-dev libzstd-dev
+        pip install meson ninja zstandard
+
+    # TODO: turn -Dwerror back on when build is fixed
+    - name: Configure (Meson)
+      run: |
+        meson setup build/ \
+          --prefix=$(pwd)/build-release \
+          --native-file scripts/ci/linux-x86_64-build-test-ci.ini
+
+    - name: Build & Install
+      run: |
+        meson compile -C build/
+        meson install -C build/
+
+    - name: Run Test
+      run: $(pwd)/build-release/bin/taisei -R $(pwd)/misc/tests/test-replay.tsr
       env:
         TAISEI_NOPRELOAD: 0
         TAISEI_PRELOAD_REQUIRED: 1
 
     - name: Upload Log
       uses: actions/upload-artifact@v2
-      if: failure()
       with:
-        name: taisei_linux_fail_log
+        name: taisei_linux_release.log
```

Comment:
Why is it called "release" now? That's a bit confusing since this doesn't make release builds.